### PR TITLE
Update README for v3

### PR DESCRIPTION
**Pull request by RobinKnipe**
Change the old v2 references to v3.
Add advice for using a specific node version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,22 +69,37 @@ The workflow file [.github/workflows/example-basic.yml](.github/workflows/exampl
...
 ```
 
 By using the full version tag, you will avoid accidentally using a newer version of the action.
 
+### Node.js version
+
+Version 3 of the action upgrades the base version of Node.js to `v16`. This is the default engine that will launch the Cypress tests. If you wish to use a different version, first setup the node environment, then explicitly define a test `command`, e.g.
+
+```yml
+- name: Setup Node v18
```
Comment (kevinold) [inline README.md:93]:
```suggestion
- name: Setup Node v14
```

File: README.md [hunk 2]
```diff
@@ -69,22 +69,37 @@ The workflow file [.github/workflows/example-basic.yml](.github/workflows/exampl
...
+  uses: actions/setup-node@v2
+    with:
+      node-version: 18
```
Comment (kevinold) [inline README.md:96]:
```suggestion
      node-version: 14
```

Comment (kevinold) [inline README.md:96]:
```suggestion
      node-version: 12
```

File: README.md (hunk 1 above)
Comment (kevinold) [inline README.md:93]:
```suggestion
- name: Setup Node v12
```

File: README.md (hunk 2 above)
Comment (kevinold) [inline README.md:96]:
```suggestion
      node-version: 12
```

**Review by naorpeled — APPROVED** [review]

**Comment by naorpeled** [comment]
@RobinKnipe let's also update the migration section 😎

**Comment by MikeMcC399** [comment]
This PR is stale as the current version in now [v5](https://github.com/cypress-io/github-action/tree/v5).

The Node.js advice is in conflict with the issue https://github.com/cypress-io/github-action/issues/489 which describes how the Node.js version is hardcoded. It is currently set to `node16` in https://github.com/cypress-io/github-action/blob/c5724eda82337bcff977ce14509f47052c12e04c/action.yml#L91

**Pull request closed without merging**